data/hai/hai reveal 1 intro.txt

```diff
@@ -1053,6 +1053,8 @@ mission "Hai Leaks Response 6"
 	description "Sayari's friend thinks someone with legal protection is kidnapping Hai. Check spaceports in nearby parts of human space for evidence of a Hai trafficking operation."
 	autosave
 	source "Hai-home"
+	destination "Hai-home"
+	# Per the wiki: Explicitly listing destination may be needed for the clearance tag to work.
```

Comment:
Are you sure about this?
It doesn't look to me like the wiki says this and the code definitely doesn't indicate this.
Creating a test mission for myself, the clearance works perfectly fine when no destination is specified and allowed to default to the source.

Reply:
And yet, with this mission active, multiple people report being unable to land on Hai-home.

So idk what the deal is, but if it's not this then it's a bug that exists well outside of my scope.

Reply:
Do you have save files?

Reply:
Erm, it might be testable with this:

[Cheaty.McCheater.1006.free.worlds.done~GO PIRATE GO.txt](https://github.com/endless-sky/endless-sky/files/10702705/Cheaty.McCheater.1006.free.worlds.done.GO.PIRATE.GO.txt)

Reply:
I don't have a proper save though that I think is useful for it - @roadrunner56 reported this issue though so they might

Reply:
Hmm, I don't have the issue anymore, but I can try using the save file you provided.